packages/@aws-cdk/aws-appmesh/lib/virtual-service.ts

```diff
@@ -177,11 +183,9 @@ interface VirtualServiceAttributes {
   /**
    * The name of the service mesh that the virtual service resides in
    *
-   * Used to derive ARN from mesh name if ARN not provided
```

Comment:
Same comment here (remove `virtualServiceArn` from `VirtualServiceAttributes`, make `virtualServiceName` and `mesh` required).